### ISSUE TITLE
question about new label 

### ISSUE THREAD

**Issue by carry-xz** [kaiwang960112/AFM#1]
hi，nice work! I found that in your code， you didnot calcuate the new soft label as in parper, why?
loss = args.weight_naive * criterion(scores, labels) + args.weight_afm * criterion(scores_naive, labels)

**Comment by kaiwang960112** [comment]
hello, thank you for your advice. we will check our code again and update it recently.

**Comment by hwei-cs** [comment]
any update?
